packages/common/src/failure.ts

```diff
@@ -261,6 +285,12 @@ export interface ApplicationFailureOptions {
    * Cause of the failure
    */
   cause?: Error;
+
+  /**
+   * Severity category of the application error.
+   * Maps to corresponding client-side logging/metrics behaviors.
```

Comment:
Not sure what is meant by "corresponding **client-side** logging/metrics behaviors".

Reply:
As in not metrics reported by server

Reply:
I see. What about this?

```suggest
Affects worker-side logging and metrics behaviors of this failure.
```